### ISSUE TITLE
cmder/conemu usage

### ISSUE THREAD

**Issue by PabloSzx** [Schniz/fnm#452]
I'm currently using [cmder](https://cmder.net) with the new [Microsoft terminal](https://github.com/microsoft/terminal), and I'm pretty sure is the best environment for development with Windows (in my opinion, after testing a lot of different ones), but I'm not being able to use fnm with it's "fnm env", any hints or anything from anyone that knows anything about it?

**Comment by Schniz** [comment]
What do you mean by not being able to use it? It is running CMD or PowerShell underneath, no?

**Comment by PabloSzx** [comment]
> What do you mean by not being able to use it? It is running CMD or PowerShell underneath, no?

It doesn't use powershell, and it works by injecting into cmd.exe

**Comment by Schniz** [comment]
Have you enabled developer mode in windows? (Not a Windows user myself so I hope I'll be able to assist)

**Comment by PabloSzx** [comment]
> Have you enabled developer mode in windows? (Not a Windows user myself so I hope I'll be able to assist)

yes..

**Comment by Schniz** [comment]
Maybe you can try using the Linux build? Would it work on Conemu?

**Comment by Lunchb0ne** [comment]
From what I understand, the command,

```cmd
FOR /f "tokens=*" %i IN ('fnm env --use-on-cd') DO CALL %i
```
Basically tells the shell to execute `fnm env --use-on-cd` and then parse it to set the different env variables, that gets spit out of the `fnm`command, as cmder provides a user_profile.cmd (`%CMDER_ROOT%/config/user_profile.cmd`) file which runs at the start and executes anything you put in it. 
As everything happens to be called from inside cmd, you have to slightly change up how the command is written and use `%%` instead of just `%`.

You can do the following:
```cmd
:: %CMDER_ROOT%\bin\fnm_init.cmd
@echo off
FOR /f "tokens=*" %%z IN ('fnm env --use-on-cd') DO CALL %%z
```

And then you can append the following to the CMDER user_profile
```cmd
:: %CMDER_ROOT%\config\user_profile.cmd
:: if you have the init file in \bin
call "%CMDER_ROOT%\bin\fnm_init.cmd"
```

Doing this should let you use fnm normally.

**Comment by Schniz** [comment]
Cool -- feel free to open a PR that allows that usage 🙏 
Windows Terminal should work fine (this is what I tested it on) 😃  but I'm not a Windows user

**Comment by Lunchb0ne** [comment]
If I get more time I'll try adding completions support for Cmder/Clink.

**Comment by Schniz** [comment]
Looks like this issue can be closed thanks to #493. Thanks @Lunchb0ne 😃
